components/rewards/PointsHistory.js

```diff
@@ -36,10 +37,14 @@ function PointsHistory({ transactions, user, updates, navigation }) {
         ) : (
           <Text />
         )}
+        {/* end receipt prompt */}
         {/* Points history */}
         <View>
           <ScrollView style={{ marginTop: 50 }}>
-            <Text style={{ textAlign: 'center' }}> Recent Transactions</Text>
+            <Caption style={{ textAlign: 'center' }}>
```

Comment:
In our effort to reduce in-line styling, you could create a container to align things to the left! Especially since all other captions (according to the Figma) should be left-aligned as well. This could done as another ticket too